plenum/common/ledger_manager.py

```diff
@@ -315,6 +338,9 @@ def processLedgerStatus(self, status: LedgerStatus, frm: str):
                             format(self, ledgerInfo.ledgerStatusOk, ledgerId))
                 logger.info('{} found from ledger status {} that it does not need catchup'.
                             format(self, ledgerStatus))
+                if ledgerId in self.request_ledger_status_action_ids:
+                    self._cancel(self.request_ledger_status_action_ids[ledgerId])
```

Comment:
We should also cancel re-asking for last ConsistencyProofs at this moment.

Reply:
Done.